### PR TITLE
[IMP] l10n_mx: Removed all accounts, only leave the accounts that are used in data

### PR DESCRIPTION
**Pull request by luistorresm**
[IMP] l10n_mx: Removed all accounts, only leave the accounts that are used in data

Added server action that set account tag in each account when
this is created or updated, this server action search account tag
with the same code and assign the parent and corresponding tag.

Added nature field and data in tags to allow set this value in
electronic account report, and set this value with data data

- [ ] Fix https://github.com/Vauxoo/mexico/issues/329#issuecomment-258080569

### CODE REVIEW

File: addons/l10n_mx/data/l10n_mx_chart_data.xml [hunk 1]
```diff
@@ -1,6596 +1,133 @@
...
-        <field name="code">1129003000</field>
+    <record id='cuenta102_01' model='account.account.template'>
+        <field name='name'>Bancos nacionales</field>
+        <field name='code'>102.01.01</field>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
         <field name="reconcile" eval="True"/>
-        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="tag_ids" eval="[(6,0,[ref('account_tag_102'), ref('account_tag_102_01')])]"/>
     </record>
     <record id="vauxoo_mx_chart_template" model="account.chart.template">
```
Comment (luistorresm) [inline addons/l10n_mx/data/l10n_mx_chart_data.xml]: Remove Vauxoo

File: addons/l10n_mx/data/action_server_data.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+object.assign_account_tag()
+            </field>
+        </record>
+
+        <record model="base.action.rule" id="action_rule_create_assign_account_tag">
+            <field name="name">Assign Account Tag</field>
+            <field name="model_id" eval="ref('account.model_account_account')"/>
+            <field name="sequence">0</field>
+            <field name="active">True</field>
+            <field name="kind">on_create</field>
```
Comment (luistorresm) [inline addons/l10n_mx/data/action_server_data.xml]: Es correcto solo on_crate @nhomar ?

File: addons/l10n_mx/data/l10n_mx_chart_data.xml [hunk 3]
```diff
@@ -1,6596 +1,119 @@
...
-            <field name='name'>IVA acreditable de importación pagado</field>
-            <field name='code'>118.02</field>
+        <record id='cuenta216_13' model='account.account.template'>
+            <field name='name'>Impuestos retenidos de iva efectivamente pagados</field>
+            <field name='code'>216.13.01</field>
             <field name="chart_template_id" ref="vauxoo_mx_chart_template"/>
-            <field name="user_type_id" ref="account_type_other"/>
-            <field name="tag_ids" eval="[(6,0,[ref('account_tag_118'), ref('account_tag_118_02')])]"/>
+            <field name="user_type_id" ref="account.data_account_type_current_liabilities"/>
+            <field name="tag_ids" eval="[(6,0,[ref('account_tag_216_10')])]"/>
```
Comment (luistorresm) [inline addons/l10n_mx/data/l10n_mx_chart_data.xml]: FYI Es correcto que se asigne este tag, porque el código de la cuenta no corresponde a el código del tag.

File: addons/l10n_mx/data/action_server_data.xml (hunk 2 above)
Comment (nhomar) [inline addons/l10n_mx/data/action_server_data.xml]: on write creo que también.

**Pull request merged**